src/blockchain_txn_reduce_validator_stake_v1.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+package helium;
+
+message blockchain_txn_reduce_validator_stake_v1 {
```

Comment:
This is unused? If so we should probably remove the definition. Otherwise it's going to need a signature from the owner?

Reply:
removed